### PR TITLE
Added install commands for cmake so find_package will work

### PR DESCRIPTION
**Pull request by RobertoRoos**
Solves #25 

Other targets should now be able to be built out of the box.

Note: I mimicked the code from https://github.com/RealTimeBiomechanics/Concurrency, just because I happened to have that one open.

I did reformat the cmake file, I hope that's okay.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,64 +1,70 @@
...
-)
-target_include_directories( ${PROJECT_NAME} 
-    PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
-)
-target_link_libraries( ${PROJECT_NAME}
-    PRIVATE $<$<STREQUAL:"${CMAKE_CXX_COMPILER_ID}","MSVC">:ws2_32.lib>
-)
-# Enable C++11
+target_link_libraries(${PROJECT_NAME}
+        PRIVATE $<$<STREQUAL:"${CMAKE_CXX_COMPILER_ID}","MSVC">:ws2_32.lib>)
```
Comment (helmesjo) [inline CMakeLists.txt:17]: This can be made public instead (and the note in the readme can then be removed).

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,64 +1,70 @@
...
+project(qualisys_cpp_sdk)
 
 option(BUILD_EXAMPLES "Build examples" OFF)
 
-project(qualisys_cpp_sdk)
+add_library(${PROJECT_NAME}
+        Markup.cpp
+        Network.cpp
+        RTPacket.cpp
+        RTProtocol.cpp)
```
Comment (helmesjo) [inline CMakeLists.txt:10]: Please avoid changing the convention (have the `)` on a new line). :)

File: CMakeLists.txt (hunk 1 above)
Comment (RobertoRoos) [reply CMakeLists.txt]: Oh yeah, that makes sense. Updated.

**Comment by helmesjo** [comment]
Thank you for the contribution! :)

**Comment by RobertoRoos** [comment]
Thanks. I updated based on your comments, let me know if I should squash the commits.

**Review by EmilRosenquist — APPROVED** [review]

**Review by helmesjo — APPROVED** [review]

**Comment by helmesjo** [comment]
> Thanks. I updated based on your comments, let me know if I should squash the commits.

No problem, I'll do a squash merge.

**Pull request merged**